util/mknum.pl

```diff
@@ -138,8 +138,9 @@
     }
     if ($dropped) {
         print STDERR "${ordinals_file}: Dropped $dropped new symbols\n";
+        $stats{unassigned} = 0 unless defined $stats{unassigned};
+        $unassigned = $stats{unassigned} - $dropped;
     }
-    $unassigned = $stats{unassigned} - $dropped;
```

Comment:
That looks a bit clumsy.  How about this?

``` perl
    if ($dropped) {
        print STDERR "${ordinals_file}: Dropped $dropped new symbols\n";
    }
    $unassigned = $stats{unassigned} // 0 - $dropped;
```

Reply:
I find my change to be more clear, even if it is less perlish magic.

Reply:
Uhmm, `//` is a perfectly normal perl operator, there's really nothing magic about it.

(I suppose you know this bashism:  `${foo:-something}` ...  that's about as magic that it is)

Reply:
https://www.in-ulm.de/~mascheck/bourne/v7/  I know `${foo?bar}` etc.  Never used bash much.  I prefer "rc"

Reply:
Not bashism.  POSIX sh.